engine/polyglot-api/src/main/java/org/enso/polyglot/HostAccessFactory.java

```diff
@@ -4,11 +4,6 @@
 
 /** Utility class for creating HostAccess object. */
 public class HostAccessFactory {
-
```

Comment:
Taken from #9201, which will be merged before this PR.

Reply:
btw. not strictly necessary but I recommend the [stacked PR workflow](https://stacking.dev/) in such case, it makes stuff clearer.

You could rebase this PR on top of #9201, and change it's base to be #9201's branch - this way this change 'borrowed' from other PR would not show in the diff. Once the 'parent' PR is merged, GH automatically changes base to its parent PR (i.e. `develop`) (although that usually means another rebase is needed).

IMHO it makes stuff cleaner and I use this workflow _a lot_
![image](https://github.com/enso-org/enso/assets/1436948/fd6cc615-ddcf-485f-914a-71c31e8a70b3)

Ofc it's a bit subjective, so not sure if you'd find such a workflow easier, just sharing as my perspective.